### ISSUE TITLE
Unchecked checkbox has aria-checked="null" when used with Flow

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-checkbox#163]
LightHouse report for https://bakery-flow.demo.vaadin.com/storefront

> [aria-*] attributes do not have valid values

```html
<vaadin-checkbox class="checkbox mobile" tabindex="0" aria-checked="null" role="checkbox">Show past orders</vaadin-checkbox>
```

This only happens when using `vaadin-checkbox` with Flow in Bakery.

Should be easy to fix by changing this line to use `Boolean(checked)`

https://github.com/vaadin/vaadin-checkbox/blob/7572a685d5c21ccc31322d29a048ed17e154fe94/src/vaadin-checkbox.html#L225
